src/src/Entity/BlogEntity.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\BlogEntityRepository;
+use Doctrine\DBAL\Types\Types;
+use Doctrine\ORM\Mapping as ORM;
+
+#[ORM\Entity(repositoryClass: BlogEntityRepository::class)]
+class BlogEntity
```

Comment:
`BlogPost` might be a more fitting name, but `BlogEntity` is not exactly wrong either.